src/main/resources/application.conf

```diff
@@ -199,32 +199,28 @@ mantis {
   }
 
   consensus {
-    # FIXME this was moved here from the `mining` section
     active-timeout = 5 seconds
```

Comment:
in the current form the meaning of this config is really confusing: `consensus.active-timeout` - sounds like there's a timeout for consensus or something. As it's only used for hashrate reporting in one of RPC apis, maybe `rpc` config is a good place to put it.

Reply:
This was in the `mining` section, so according to the original idea it seems to belong to the current `ethash` section. I must have moved it from `mining` to the generic `consensus` and forgot to re-place it under `ethash` while refactoring. 

Given the above, can you both confirm that a better place indeed is `rpc`?

Reply:
I'd still go for `rpc`

Reply:
@rtkaczyk ?

Reply:
Yes, `rpc`

Reply:
[Moved](https://github.com/input-output-hk/mantis/pull/410/commits/3282958dbbdf135454ce0805bb69aed710e6afc1).